src/seedu/addressbook/Main.java

```diff
@@ -78,15 +78,26 @@ private void exit() {
         System.exit(0);
     }
 
-    /** Reads the user command and executes it, until the user issues the exit command.  */
+    /** Reads the user command and executes it, until the user issues the exit command.
+     *
+     * I referenced PR #133 to use similar logic in handling the exceptions being thrown throughout the methods.
```

Comment:
Ideally you can include this information while creating the PR. Comments in code should be about the code.